.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,7 +6,17 @@ about: Create a report to help reproduce and fix the issue
 
 ## Issue Summary
 
-<!-- Briefly explain your issue in a few plain sentences, e.g. "X has Y issue" "When I do X, Y happens instead of Z" "X on Y map has Z issue". You may copy and paste the issue title here if it is suitable. -->
+<!-- Briefly explain your issue in a few plain sentences, e.g. "X has Y issue" "When I do X, Y happens instead of Z" "X on Y map has Z issue". You may copy and paste the issue title here if it is suitable. Screenshots can be especially helpful. -->
+
+<!-- Please help cull duplicate and confusing issues. Your answers here are not required unless specifically marked as such. -->
+
+## Checklist
+
+[] Yes, I have searched for this issue. (REQUIRED)
```

Comment:
A significant amount of the people using this template are doing it from the in game Report Issue button. It's significantly harder to search in that environment. I don't know that this is a big enough problem that we need to be so scary about it.